docs/en/stack/ml/df-analytics/flightdata-regression.asciidoc

```diff
@@ -0,0 +1,367 @@
+<7> Specifies a memory limit for the job. If the job requires more than this 
+amount of memory, it fails to start. This makes it possible to prevent job 
+execution if the available memory on the node is limited.
+
+
+The API returns the following response:
+
+[source,console-result]
+--------------------------------------------------  
+{
+  "id" : "model-flight-delays",
+  "source" : {
+    "index" : [
+      "kibana_sample_data_flights"
+    ],
+    "query" : {
+      "range" : {
+        "DistanceKilometers" : {
+          "gt" : 0
+        }
+      }
+    }
+  },
+  "dest" : {
+    "index" : "df-flight-delays",
+    "results_field" : "ml"
+  },
+  "analysis" : {
+    "regression" : {
+      "dependent_variable" : "FlightDelayMin",
+      "training_percent" : 90.0
+    }
+  },
+  "analyzed_fields" : {
+    "includes" : [ ],
+    "excludes" : [
+      "Cancelled",
+      "FlightDelay",
+      "FlightDelayType"
+    ]
+  },
+  "model_memory_limit" : "100mb",
+  "create_time" : 1571918329132,
+  "version" : "7.5.0",
+  "allow_lazy_start" : false
+}
+--------------------------------------------------
+--
+
+. Start the job.
++
+--
+Use the {ref}/start-dfanalytics.html[start {dfanalytics-jobs}] and 
+{ref}/stop-dfanalytics.html[stop {dfanalytics-jobs}] APIs to start and stop 
+jobs. 
+
+[source,console]
+--------------------------------------------------
+POST _ml/data_frame/analytics/model-flight-delays/_start
+--------------------------------------------------
+// TEST[skip:TBD]
+
+
+The job takes a few minutes to run. Runtime depends on the local hardware and 
+also on the number of documents and fields that analyzed. The more fields and 
+documents, the longer the job to run.
+--
+
+. Check the job stats to follow the progress by using the 
+{ref}/get-dfanalytics-stats.html[get {dfanalytics-jobs} statistics API].
++
+--
+
+
+[source,console]
+--------------------------------------------------
+GET _ml/data_frame/analytics/model-flight-delays/_stats
+--------------------------------------------------
+// TEST[skip:TBD]
+
+
+The API call returns the following response: 
+
+[source,console-result]
+----  
+{
+  "count" : 1,
+  "data_frame_analytics" : [
+    {
+      "id" : "model-flight-delays",
+      "state" : "stopped",
+      "progress" : [
+        {
+          "phase" : "reindexing",
+          "progress_percent" : 100
+        },
+        {
+          "phase" : "loading_data",
+          "progress_percent" : 100
+        },
+        {
+          "phase" : "analyzing",
+          "progress_percent" : 100
+        },
+        {
+          "phase" : "writing_results",
+          "progress_percent" : 100
+        }
+      ]
+    }
+  ]
+}
+----  
+
+
+The job has four phases. When all the phases have completed, the job 
+state becomes `stopped` and the results are ready to view and evaluate.
+--
+
+
+[[flightdata-regression-results]]
+==== Viewing results
+
+. Run the following command to view the results in the destination index:
```

Comment:
I think it is worth mentioning that the results are stored in a "normal" elasticsearch index.
e.g. `Use the standard elasticsearch search command to view the results in the destination index`